google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1beta2/StreamWriterV2.java

```diff
@@ -137,6 +140,7 @@ private StreamWriterV2(Builder builder) throws IOException {
     this.streamName = builder.streamName;
     this.maxInflightRequests = builder.maxInflightRequest;
     this.maxInflightBytes = builder.maxInflightBytes;
+    this.traceId = builder.traceId;
```

Comment:
let's start with the user trace id/job id here?

Reply:
It is supposed to be general, so I don't want to limit it to job id. I feel like trace id is a more general term here.

Reply:
This will be used for all use cases, not just dataflow. So I would prefer to more general term like trace id here.